perfkitbenchmarker/linux_benchmarks/spark_benchmark.py

```diff
@@ -31,6 +31,13 @@
 job was received by the platform and when it ran), and a runtime, which is
 the time the job took to run, as reported by the underlying cluster.
 
+This benchmark can be used to run a hadoop job by setting the spark_job_type
+flag to hadoop, eg:
```

Comment:
If you really want to reuse this for Hadoop jars (without changing all the naming), I would make it clear that Hadoop support is secondary.

can also be used to run an Apache Hadoop MapReduce job, if the cluster supports that (has MapReduce libraries and runs Hadoop YARN).